scripts/ci/ensure-all-node-requirements-available.sh

```diff
@@ -62,6 +63,9 @@ git ls-tree -r HEAD | while IFS='' read line; do
               requirement_found=1
             elif [[ -f "$localpath.js" ]] || [[ -f "$localpath/index.js" ]]; then
               requirement_found=1
+            # Webpack HTML loader
+            elif [[ "$localpath" =~ $HTML_REGEX ]]; then
+              requirement_found=1
```

Comment:
IMHO it would be better if this actually checked if the relevant HTML file exists. I did a bit of local experimentation, and the below diff adds this feature:
```diff
--- a/scripts/ci/ensure-all-node-requirements-available.sh
+++ b/scripts/ci/ensure-all-node-requirements-available.sh
@@ -49,7 +49,7 @@ git ls-tree -r HEAD | while IFS='' read line; do
     extension=${filename##*.}
     if [[ "$extension" == "js" ]]; then
       # 'grep -v' to filter out any comment-blocks
-      grep 'require(' "$fullpath" | grep -v "^ \* " | while IFS='' read line; do
+      grep 'require(' "$fullpath" | grep -v -E "^ +\* " | while IFS='' read line; do
         if [[ "$line" =~ $REQUIRE_REGEX ]]; then
           required=${BASH_REMATCH[1]}
         fi
@@ -64,7 +64,7 @@ git ls-tree -r HEAD | while IFS='' read line; do
             elif [[ -f "$localpath.js" ]] || [[ -f "$localpath/index.js" ]]; then
               requirement_found=1
             # Webpack HTML loader
-            elif [[ "$localpath" =~ $HTML_REGEX ]]; then
+            elif [[ "$localpath" =~ $HTML_REGEX ]] && [[ -f "$localpath" ]]; then
               requirement_found=1
             fi
           else
```
:smiley:

Reply:
Cool, thanks! I updated the PR with your fix :)